src/bindings/python/src/pyopenvino/core/common.cpp

```diff
@@ -33,6 +33,8 @@ const std::map<ov::element::Type, py::dtype>& ov_type_to_dtype() {
         {ov::element::u4, py::dtype("uint8")},
         {ov::element::nf4, py::dtype("uint8")},
         {ov::element::i4, py::dtype("int8")},
+        {ov::element::f8e4m3, py::dtype("uint8")},
+        {ov::element::f8e5m2, py::dtype("uint8")},
```

Comment:
@jiwaszki Just wanted to double check, should this map be extended anyway (as there is no numpy representation for f8 types). This is optional change, the required part to enable providing the new fp8 types to Constant or Convert is only binding to `ov.Type`:
https://github.com/openvinotoolkit/openvino/blob/d1f89dead52a11b08a8dc76a5c670e8328546b7f/src/bindings/python/src/pyopenvino/graph/types/element_type.cpp#L52-L53